EAD/views/shelf1.ejs

```diff
@@ -54,20 +54,20 @@
                           <div>
                               <!-- <div class="af-explames-demo-mask"></div> -->
                               <!-- <canvas width="140" height="185" style="position: absolute; width: 280px; height: 370px; left: -65px; top: -92.5px; z-index: 5;"></canvas> -->
-                              <% if(owner[i].paid === 0) {%>
+                              <% if(owner[i].owner === '1' && owner[i].paid !== 1) {%>
                                 <form action="/shelf/charge" method="post">
                                   <script
                                   src = "https://checkout.stripe.com/checkout.js"
                                   class = "stripe-button"
-                                  data-key = "pk_test_d8zWt8DzfYfP9i9WIwMwJJh900xIyYtz5s"
-                                  data-amount = "2000"
+                                  data-key = "pk_test_51Gtt6XLG3iR3WbW87jLGZqAHKPibjAmJeK4dqHoVPVsGMA2RVOP1Vjk2GYYgpOTUtTOSGAAXzQKUx5RKZvETBfkC00mH6pT7Wp"
+                                  data-amount = "<%= owner[i].amount*100 %>"
```

Comment:
why are you showing amount here, what ever he pays here it will be stored in amount field ryt?

Reply:
whatever amount the book owner requests the purchaser will be shown here, and the purchaser needs to pay that much amount of money to unlock it.